7.chat-plugin/client.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Socket.IO chat</title>
```

Comment:
the logic in this file seems fine :) 
I would have organized the code a bit tho, maybe add some comments to make it more organized. 

-- this is not something you have to do. but just keep it in mind 😉